cmake/platform.cmake

```diff
@@ -428,6 +428,8 @@ elseif(UNIX OR MINGW)
             # For native compilation tune for the host processor
             if (CMAKE_SYSTEM_PROCESSOR STREQUAL CMAKE_HOST_SYSTEM_PROCESSOR)
                 append(DEF_ARCH_OPT_FLAGS "-mcpu=native")
+            else()
+                append(DEF_ARCH_OPT_FLAGS "-mcpu=power10 -mmma")
```

Comment:
just for my own knowledge, could you elaborate what exactly is going wrong? 
also, could we add a comment here to explain the addition?

Reply:
A comment would definitely help future IBM developers.